src/wayland/seat/mod.rs

```diff
@@ -313,6 +317,59 @@ impl Seat {
         }
     }
 
+    /// Adds the touch capability to this seat
+    ///
+    /// You are provided a [`TouchHandle`], which allows you to send input events
+    /// to this pointer. This handle can be cloned.
+    ///
+    /// Calling this method on a seat that already has a touch capability
+    /// will overwrite it, and will be seen by the clients as if the
+    /// touchscreen was unplugged and a new one was plugged in.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # extern crate wayland_server;
+    /// #
+    /// # use smithay::wayland::seat::Seat;
+    /// #
+    /// # let mut display = wayland_server::Display::new();
+    /// # let (mut seat, seat_global) = Seat::new(
+    /// #     &mut display,
+    /// #     "seat-0".into(),
+    /// #     None
+    /// # );
+    /// let touch_handle = seat.add_touch();
```

Comment:
This example doesn't really explain anything more than the function docs itself.

If the seat creation are made visible it would be more jusitifed to have.

Reply:
Correct, but I'd say the same applies for the existing code examples for keyboard/pointer?

> If the seat creation are made visible it would be more jusitifed to have.

What do you mean by this? Removing the `#` before the seat stuff?